lib/Core/ExternalDispatcher.cpp

```diff
@@ -89,7 +94,12 @@ ExternalDispatcher::ExternalDispatcher() {
   dispatchModule = new Module("ExternalDispatcher", getGlobalContext());
 
   std::string error;
+#if LLVM_VERSION_CODE >= LLVM_VERSION(3, 6)
+  dispatchModule_uniptr.reset(dispatchModule);
+  executionEngine = EngineBuilder(std::move(dispatchModule_uniptr)).create();
```

Comment:
You can also chain in `setErrorStr` before the call to `create` to provide an error string. Otherwise for LLVM versions >= 3.6 if the `EngineBuilder` fails then the "unable to make jit" error message will be empty.